### ISSUE TITLE
Project status - abandoned?

### ISSUE THREAD

**Issue by neutmute** [facultymatt/angular-unsavedChanges#63]
Hi @facultymatt - looks like some great work here but I'm hesitant to use in a build as it looks abandoned based on open issues/PRs that haven't been responded to.

Is there any intention to pick it back up?

**Comment by ctysingh** [comment]
It is a very useful utility however I tool feel not so comfortable in picking up this since there has not been any updates from a long time. I wish angular team to add this feature in their core library.

**Comment by simonv3** [comment]
Is there something that people use as an alternative that is maintained?

**Comment by neutmute** [comment]
I ended up using this and it worked ok - YMMV :|

**Comment by simonv3** [comment]
Yeah, we've been using it for about 2 years, but just needed a small change (https://github.com/facultymatt/angular-unsavedChanges/pull/72), but I don't think this will get merged in, so we're running off of our own version of this library for now. But! I don't really have the capacity to maintain it, so was hoping there was something that was actively maintained out there.
